src/js/components/Anchor/doc.js

```diff
@@ -103,4 +103,6 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
+  ...themeDocUtils.focusStyle,
+  ...themeDocUtils.normalizeColor('The text color used inside the Anchor.'),
```

Comment:
I don't think this is accurate, `Anchor` uses `normalizeColor` only for the icon color and not for the text.
https://github.com/grommet/grommet/blob/726f4f4abdb8050f1de70a9b5c69793aadb628a0/src/js/components/Anchor/Anchor.js#L46

`normalizeColor` is auto generating 
```
'global.colors.text': {
      description,
      type: 'string | { dark: string, light: string }',
      defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
    }, 
```
and `'global.colors.text'` isn't being used in anchor at all.